tron/scheduler/common.py

```diff
@@ -0,0 +1,23 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+import datetime
+import logging
+import random
+
+from tron.utils import timeutils
+
+log = logging.getLogger(__name__)
```

Comment:
Do we really want this in `common`, as opposed to different loggers per scheduler? This way, the logger will always be 'tron.scheduler.common'. It's not used right now anyway.

Reply:
yeah, i kinda overlooked that, will make a separate logger in each scheduler module